source/blog/036-sass-and-native-nesting.md

```diff
@@ -0,0 +1,145 @@
+browser to add support for the new [native CSS nesting feature]. This feature—
+inspired by Sass's nesting—adds the ability to nest style rules in plain CSS,
+and even uses Sass's convention of `&` to refer to the parent selector.
+
+[native CSS nesting spec]: https://drafts.csswg.org/css-nesting/
+
+We here at Sass HQ are honored every time our language design inspires
+improvements in CSS itself. We're excited to see the usability and clarity
+benefits of nesting brought to even more CSS authors as more browsers continue
+to roll out support for this feature.
+
+## The Future of Sass Nesting
+
+This raises an important question, though: what will happen to Sass's nesting?
+First of all, we won't ever change existing valid Sass code so that it starts
+emitting CSS that's incompatible with widely-used browsers. This means that even
+if we did decide to phase out Sass nesting and just emit plain CSS nesting
+instead, we wouldn't do so until [98% of the global browser market share]
+supported native nesting.
+
+[98% of the global browser market share]: https://github.com/sass/dart-sass#browser-compatibility
+
+More importantly, though, **native CSS nesting is subtly incompatible with Sass
+nesting**. This effects three different cases:
+
+1. Native CSS nesting implicitly wraps the parent selector in [`:is()`], while
+   Sass copies its text into the resolved selector. That means that
+
+   [`:is()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/:is
+
+   ```scss
+   .foo, #bar {
+     .baz { /* ... */ }
+   }
+   ```
+
+   produces the selector `.foo .baz, #bar .baz` in Sass but `:is(.foo, #bar)
+   .baz` in native CSS. This changes the specificity: `:is()` always has the
+   specificity of its _most specific selector_, so `:is(.foo, #bar) .baz` will
+   match
+   
+   ```html
+   <div class=foo>
+     <p class=bar>
+   </div>
+   ```
+   
+   with specificity 1 0 1 in native CSS and 0 0 2 in Sass even though neither
+   element is matched by ID.
+
+2. Also because native CSS nesting uses `:is()`, a parent selector with
+   descendant combinators will behave differently.
+
+   ```scss
+   .foo .bar {
+     .green-theme & { /* ... */ }
+   }
+   ```
+
+   produces the selector `.green-theme .foo .bar` in Sass, but in native CSS it
+   produces `.green-theme :is(.foo .bar)`. This means that the native CSS
+   version will match
+   
+   ```html
+   <div class=foo>
+     <div class="green-theme">
+       <p class=bar>
+     </div>
+   </div>
+   ```
+
+   but Sass will not, since the element matching `.foo` is outside the element
+   matching `.green-theme`.
+
+3. Native CSS nesting and Sass nesting both support syntax that looks like
+   `&foo`, but it means different things. In native CSS, this adds a type
+   selector to the parent selector, so
+   
+   ```scss
+   .foo {
+     &div { /* ... */ }
+   }
+   ```
+
+   produces the selector `div.foo`. But in Sass, this adds a suffix to the
```

Comment:
I think you should add what Sass would do here: `.foodiv` to highlight how different it behaves. The example below does show that, but I think it's not clear enough because the inputs are different